File: src/counter.py
class Counter:
    def __init__(self, name):
        self.__name = name
        self.__count = 0

    def click(self):
        self.__count += 1

    def count(self):
        return self.__count

    def __str__(self):
        return self.__name + \
            "'s count is " + \
            str(self.__count)

File: src/Counter.java
public class Counter {
    private String name;
    private int count = 0;

    public Counter(String name) {
        this.name = name;
    }

    public void click() { ++count; }

    public int getCount() { return count; }

    public String toString() {
        return name + "'s count is " + count;
    }
}
